### PR TITLE
Add Knative v1.11.0 release blog

### PR DESCRIPTION
**Pull request by dsimansk**
Changes 

- Add Knative v1.11.0 release blog

PTAL and please double check your names for spelling & copy paste error. Thanks!

/cc @knative/technical-oversight-committee 
/cc @knative/knative-release-leads

### CODE REVIEW

**Comment by kvmware** [comment]
/lgtm
/hold
please unhold if release leads happy

**Comment by psschwei** [comment]
Just a note: the blog isn't versioned, so we don't need to cherry pick it to the release branch (it'll show up on the site as soon as this merges).

File: blog/config/nav.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ nav:
     - Blog:
       - index.md
       - Releases:
+          - releases/announcing-knative-v1-11-release.md
```
[the remark this answers is not in the document]
Comment (psschwei) [reply blog/config/nav.yml:20]: :+1: Yes it is

File: blog/docs/releases/announcing-knative-v1-11-release.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+- Added Rust templates linting into CI workflow (#1814, @andrejusc)
+- Adds `func environment` to print the current function execution environment as JSON. (#1761, @lance)
+- Cluster.local: ""
+    boson-project.org: |
+      selector:
+        func.domain: "boson-project.org"
+    boson-project.net |
+      selector:
+        func.domain: "boson-project.net" (#1690, @lkingland)
```
[the remark this answers is not in the document]
Comment (dsimansk) [reply blog/docs/releases/announcing-knative-v1-11-release.md:111]: Let me check the original, I've probably copied too much text, or missed formatting.

[the remark this answers is not in the document]
Comment (dsimansk) [reply blog/docs/releases/announcing-knative-v1-11-release.md]: Here's the fix: https://github.com/knative/docs/pull/5647/commits/adc937267ac79febeedb30b6e17e1095369e1fef

**Pull request merged**